aspnetcore/security/cors.md

```diff
@@ -131,18 +131,34 @@ For some options, it may be helpful to read the [How CORS works](#how-cors-works
 
 ### Set the allowed origins
 
-To allow one or more specific origins, call <xref:Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicyBuilder.WithOrigins*>:
+The CORS middleware in ASP.NET Core MVC has a ways to specify allowed origins:
+
+* <xref:Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicyBuilder.WithOrigins*>:  Allows specifying one or more server names. The server name including the scheme, protocol and port without any path information. `https://example.com`. The URL must be specified without a trailing slash (`/`). e.g.
```

Comment:
- Remove the extra space before "Allows"
- ~~e.g.~~
- What's the difference between "protocol" and "scheme" in that 2nd sentence?